cylc/flow/scheduler.py

```diff
@@ -16,63 +16,51 @@
 """Cylc scheduler server."""
 
 import asyncio
+import os
+import sys
+import traceback
 from collections import deque
 from contextlib import suppress
```

Comment:
You seem to have some strange isort configuration that is grouping and sorting `import` and `from * import` statements separately?

Reply:
isort config added in #6382; will rebase and sort out